content/en/integrations/guide/data-center-migration.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Datadog Data Center Migration
+kind: guide
+description: Learn how to migrate from one Datadog instance to another instance.
+further_reading:
+- link: "https://github.com/DataDog/datadog-sync-cli"
+  tag: "GitHub"
+  text: "datadog-sync-cli"
+- link: "/agent/configuration/dual-shipping/"
+  tag: "Documentation"
+  text: "Learn about Dual Shipping"
+- link: "/integrations/guide/azure-portal/"
+  tag: "Documentation"
+  text: "Managing the Azure Native integration"
+---
+
+## Overview
+
+If you are interested in migrating Datadog data centers, follow these instructions to plan your migration:
+
+1. Create a Datadog account.
+1. Migrate Datadog [resources or assets-pick one and go with it] using the `datadog-sync-cli` package and/or [Dual Shipping][4].
+1. Migrate Datadog integrations.
+1. Migrate real-time data.
+
+In a data center migration, no data is removed from your old Datadog organization—resources are being duplicated into your new Datadog organization. You have access to the old data and resources in your old Datadog organization, which expires after the retention period passes. 
+
+This guide walks you through migrating from one Datadog data center to another Datadog data center by using `datadog-sync-cli` to sync resources across your Datadog organizations and [Dual Shipping][4] to prevent data loss. 
```

Comment:
```suggestion
This guide walks you through migrating from one Datadog data center to another using the Datadog sync CLI tool (`datadog-sync-cli`) to sync resources across your Datadog organizations and [Dual Shipping][4] to prevent data loss. 
```